docs/Static-Site-Generators.md

```diff
@@ -20,6 +20,12 @@ Let's figure out how different static site generators output static files for ro
 - Default output: `/myPath/index.html`
 - Configuration: [trailingSlash option](https://docusaurus.io/docs/docusaurus.config.js#trailing-slash)
 
+## NuxtJS
+
+- Command: `nuxt generate`
+- Default output: `/myPath`
```

Comment:
```suggestion
- Default output: `/myPath.html`
```

I guess by default it outputs files with an .html extension 🤪 
Can you double-check that it's the correct file output when using `trailingSlash: false`? The doc is not explaining that.

Reply:
Oh, you're right.

Actually, it outputs a `/myPath/index.html` file.

It does the same with the default configuration (`trailingSlash: undefined`) or with `trailingSlash: false`.

Reply:
Thanks :)